docs/algos_a1.md

```diff
@@ -0,0 +1,81 @@
+# BLOB DETECTION Assignment
+
+## Introduction
+
+The supplementary python material is so that you’re familiar with the
+language itself. This tutorial is to learn about one specific library
+this team utilizes often: OpenCV and a common sensor we use in our
+projects: stereo camera.
+
+Why do we use a stereo camera?
+
+The primary reason is for depth estimation. Let’s say there is a human
+standing in the road and we want to stop before we hit him. You can
+imagine it would be useful to know how far we are from that person.
+Stereo cameras allow us to have this information. You can think of them
+as the “eyes” of a human driver.
+
+## Getting Started
+
+### Python Virtual Enviornment (venv)
+
+A common practice that most python programmers use is creating a virtual
+environment. This ensures we are running the correct version of a
+package regardless of what is already on the system.
+
+To create a virtual environment use the following command (you should
+already have python installed):
+
+`python3 -m venv venv`
+
+To activate the virtual environment:
+
+`source venv/bin/activate`
```

Comment:
```suggestion
Create a venv with the following:

`$ python3 -m venv .venv`

And then activate it with the following:

`$ source .venv/bin/activate`
```

Reply:
Usually we prefix shell commands with a `$` to signify they're to be run in a shell.